Jenkinsfile.nightly

```diff
@@ -36,6 +36,7 @@ pipeline {
         GOPATH = "${WORKSPACE}"
         TESTDIR = "${WORKSPACE}/${PROJ_PATH}/test"
         SERVER_BOX = "cilium/ubuntu"
+        BUILD_NUMBER = "${BUILD_NUMBER}"
```

Comment:
Why you did this one?  I can't fully understand the reason.

Reply:
Are all Jenkins variables available as env variables?